### PR TITLE
Fix download controls localdb tests

### PR DESCRIPTION
**Pull request by onursumer**
Fix cBioPortal/cbioportal#9608

- make sure that generic assay tab names are always sorted alphabetically.
- replace `describe.omit` with `describe.skip` since `describe.omit` is not a function
- implement a missing `studyViewChartHoverHamburgerIcon` function

### CODE REVIEW

File: src/shared/lib/GenericAssayUtils/GenericAssayCommonUtils.ts [hunk 1]
```diff
@@ -358,3 +358,21 @@ export function deriveDisplayTextFromGenericAssayType(
     }
     return derivedDisplayText;
 }
+
+export function getSortedGenericAssayTabInfo(
+    genericAssayEnrichmentProfilesGroupedByGenericAssayType: {
+        [key: string]: MolecularProfile[];
+    } = {}
+): { genericAssayType: string; linkText: string }[] {
+    return _.sortBy<{ genericAssayType: string; linkText: string }>(
```
Comment (alisman) [inline src/shared/lib/GenericAssayUtils/GenericAssayCommonUtils.ts:367]: nested lodash is super hard to read. can we use _.chain().keys().map().value() ?

File: src/pages/resultsView/comparison/ComparisonTab.tsx [hunk 2]
```diff
@@ -264,19 +264,17 @@ export default class ComparisonTab extends React.Component<
...
+                        ).map(helper => {
                             return (
                                 <MSKTab
                                     id={`${
                                         ResultsViewComparisonSubTab.GENERIC_ASSAY_PREFIX
-                                    }_${genericAssayType.toLowerCase()}`}
-                                    linkText={deriveDisplayTextFromGenericAssayType(
-                                        genericAssayType
-                                    )}
+                                    }_${helper.genericAssayType.toLowerCase()}`}
```
Comment (alisman) [inline src/pages/resultsView/comparison/ComparisonTab.tsx:276]: the name "helper" doesn't really help me understand what it is.

Comment (onursumer) [reply src/pages/resultsView/comparison/ComparisonTab.tsx:276]: good point, I'll try to find a better name.

Comment (onursumer) [reply src/pages/resultsView/comparison/ComparisonTab.tsx]: renamed as `genericAssayTabSpecs`

File: src/shared/lib/GenericAssayUtils/GenericAssayCommonUtils.ts (hunk 1 above)
Comment (onursumer) [reply src/shared/lib/GenericAssayUtils/GenericAssayCommonUtils.ts]: separated this into multiple operations to make it easier to read

**Pull request merged**